graphs/possible_bipartition.py

```diff
@@ -5,8 +5,37 @@ def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n2)
+        Space Complexity: O(n)
```

Comment:
The time complexity is O(N+E) because you visit each node once and traverse each edge. 

👍  Otherwise nice BFS solution.